students/g_rama/lesson03/mailroom.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+donors_data = [["John", [200, 20]],
+               ["Jeff", [500, 20]],
+               ["Susan", [1000, 20, 70]],
+               ["Rob", [250, 20]],
+               ["Ross", [200]]]
+
+
+def display_donors(donor_data):
+    """Display donors if user enters list"""
+    for _ in range(len(donor_data)):
+        print('{:10}'.format(*donor_data[_]))
```

Comment:
* underscore is used for something that will not be used -- however here you're actually using `_`
* if you need an item in the list then iterate over items instead of using range + access by index:
```
for donor in donor_data:
    print(donor)
```